src/components/hero.scss

```diff
@@ -95,22 +96,47 @@
         margin-bottom: 35px;
```

Comment:
the header is too big on mobile, we could reduce it to `48px`. Also, it's missing some lateral padding so that it doesn't fully against the edge :) <img width="455" alt="image" src="https://user-images.githubusercontent.com/14869087/182248275-4e9a8fd9-d601-4a8e-b392-52efc2993054.png">